src/elements/stepper/step/step.ts

```diff
@@ -134,6 +168,12 @@ class SbbStepElement extends LitElement {
     this._label = this._getStepLabel();
   }
 
+  protected override async firstUpdated(changedProperties: PropertyValues<this>): Promise<void> {
```

Comment:
we should not alter the modifiers of a inherited method. However, I see a couple of other occurrences :D One simple possibility is to use ".then"